superset/cli.py

```diff
@@ -480,7 +482,10 @@ def import_dashboards(path: str, recursive: bool, username: str) -> None:
             files.extend(path_object.rglob("*.json"))
         if username is not None:
             g.user = security_manager.find_user(username=username)
-        contents = {path.name: open(path).read() for path in files}
+            contents = {}
+            for path_ in files:
+                with open(path_) as file:
+                    contents[path_.name] = file.read()
```

Comment:
This is slightly unrelated, but as you're bumping the indentation, I wonder if we should attempt importing at all if `username is None`? And if so (=we assume there may be use cases where an anonymous user can import), shouldn't this added indentation be restored to its original place?